site/docs/guide/inline-queries.md

```diff
@@ -22,8 +22,8 @@ It allows you to listen for specific inline queries that match strings or regula
 If you want to handle all inline queries generically, use `bot.on('inline_query')`.
 
 ```ts
-// Shameless self-advertising in a project's documentation
-// is the best kind of advertising
+// Shameless self-advertising in one project's documentation
```

Comment:
```suggestion
// Shameless self-advertising in a project's documentation
```

Reply:
I'd say "in project's own documentation".